doc-content/enterprise-only/openshift/rhpam72-prod-ref.adoc

```diff
@@ -33,8 +33,9 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`KIE_SERVER_PWD` | `KIE_SERVER_PWD` | KIE server password (Sets the org.kie.server.pwd system property) | `${KIE_SERVER_PWD}` | False
 |`IMAGE_STREAM_NAMESPACE` | -- | Namespace in which the ImageStreams for Red Hat Middleware images are installed. These ImageStreams are normally installed in the openshift namespace. You should only need to modify this if you installed the ImageStreams in a different namespace/project. | openshift | True
 |`KIE_SERVER_IMAGE_STREAM_NAME` | -- | The name of the image stream to use for KIE server. Default is "rhpam72-kieserver-openshift". | rhpam72-kieserver-openshift | True
-|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.0". | 1.0 | True
+|`IMAGE_STREAM_TAG` | -- | A named pointer to an image in an image stream. Default is "1.1". | 1.1 | True
 |`SMART_ROUTER_HOSTNAME_HTTP` | `HOSTNAME_HTTP` | Custom hostname for http service route.  Leave blank for default hostname, e.g. <application-name>-smartrouter-<project>.<default-domain-suffix>' | `${BUSINESS_CENTRAL_HOSTNAME_HTTP}` | False
```

Comment:
Same as above, not sure what to do here. SMART_ROUTER_HOSTNAME_HTTP is not passed down to the image through an environment variable at all, it's only referenced in creating the service.

Reply:
> also I notice in the rendered documents from the JIRA that the title version has 7.3 and 7.3 zip files are referenced but the table of contents shows the name of rhpam72 templates.

Ignore this one, it's a test build artefact, the published version will have the correct version numbers